cdap-app-fabric/src/main/java/io/cdap/cdap/support/services/SupportBundleService.java

```diff
@@ -64,12 +76,20 @@
 public class SupportBundleService implements Closeable {
 
   private static final Logger LOG = LoggerFactory.getLogger(SupportBundleService.class);
-  private static final Gson GSON = new GsonBuilder().create();
+  private static final Gson GSON = ApplicationSpecificationAdapter.addTypeAdapters(new GsonBuilder())
+    .registerTypeAdapter(Arguments.class, new ArgumentsCodec())
+    .registerTypeAdapter(ProgramOptions.class, new ProgramOptionsCodec())
+    .registerTypeAdapter(org.apache.twill.internal.Arguments.class, new org.apache.twill.internal.json.ArgumentsCodec())
```

Comment:
Please make sure your imported version of Twill is the correct version (io.cdap.twill) since we may have migrated back to our own version earlier.

Reply:
right now we will use 1.0.0